eng/scripts/bomhelpers.ps1

```diff
@@ -2,310 +2,312 @@
 # Licensed under the MIT License.
 
 class MavenArtifactInfo {
-    [String] $GroupId
-    [String] $ArtifactId
-    [String] $LatestGAOrPatchVersion
-    [String] $LatestRealeasedVersion
+  [String] $GroupId
+  [String] $ArtifactId
+  [String] $LatestGAOrPatchVersion
+  [String] $LatestRealeasedVersion
 
-    MavenArtifactInfo($ArtifactId, $LatestGAOrPatchVersion, $LatestRealeasedVersion) {
-        $this.ArtifactId = $ArtifactId
-        $this.LatestGAOrPatchVersion = $LatestGAOrPatchVersion
-        $this.LatestRealeasedVersion = $LatestRealeasedVersion
-        $this.GroupId = 'com.azure'
-    }
+  MavenArtifactInfo($ArtifactId, $LatestGAOrPatchVersion, $LatestRealeasedVersion) {
+    $this.ArtifactId = $ArtifactId
+    $this.LatestGAOrPatchVersion = $LatestGAOrPatchVersion
+    $this.LatestRealeasedVersion = $LatestRealeasedVersion
```

Comment:
```suggested
  MavenArtifactInfo($ArtifactId, $LatestGAOrPatchVersion, $LatestReleasedVersion) {
    $this.ArtifactId = $ArtifactId
    $this.LatestGAOrPatchVersion = $LatestGAOrPatchVersion
    $this.LatestReleasedVersion = $LatestReleasedVersion
```

Reply:
Also we do have formal SemVer type helpers in our common ps1 scripts if you wanted to enforce validation.

Reply:
I do use SemVer while updating POM files but let these be strings here since they are likely to not be invalid.